kilted/release-rhel-build.yaml

```
@@ -97,6 +97,7 @@ package_blacklist:
 - spinnaker_camera_driver # spinnaker SDK not available as RPM package
 - spinnaker_synchronized_camera_driver # depends on spinnaker_camera_driver
 - tracetools_analysis  # jupyter-notebook has no RPM package for RHEL
+- turtle_nest # Turtle Nest does not support RHEL
```

Looks like the underlying issue is that `black` isn't available. The package built just fine before the `black` dependency.

```suggestion
- turtle_nest  # python-black has no RPM package for RHEL
```

---

Yep, that's indeed the issue!